### ISSUE TITLE
Release 0.2.0

### ISSUE THREAD

**Issue by jrbourbeau** [coiled/coiled-runtime#526]
https://github.com/coiled/coiled-runtime/pull/488 bumped package versions in preparation for a new runtime release. Given https://github.com/coiled/coiled-runtime/issues/520 we should try to release ASAP. @ncclementi are you aware of any outstanding issues?

**Comment by ncclementi** [comment]
Yes, we have a couple of problems:
1. We are seeing this constantly https://github.com/coiled/coiled-runtime/issues/468 
2. There are valid regressions and we do not know where do they come from see https://github.com/coiled/coiled-runtime/issues/500 more specifically https://github.com/coiled/coiled-runtime/issues/500#issuecomment-1314505305

**Comment by ncclementi** [comment]
Based on the comment above plan of action:
1. Do a patch release (0.1.1) of runtime pinning bokeh to avoid dashboard problems. 
2. Keep studying problems to bump all the packages for release 0.2.0 see https://github.com/coiled/coiled-runtime/issues/500 and https://github.com/coiled/coiled-runtime/issues/468

**Comment by jrbourbeau** [comment]
That sounds like a good plan to me. I'd suggest pinning `bokeh=2.4.3`

**Comment by ntabris** [comment]
this means that, e.g., test shuffle got ~30% slower (with high degree of confidence)?

**Comment by hayesgb** [comment]
@ncclementi 

As we discussed -- running another A/B test that looks at bumping everything excepting the `fsspec` packages and then making a decision seems OK.

Let's also confirm you can successfully complete a `conda build` with both of the build candidate, and the one you intend to test. 

Let's aim for a decision by midday tomorrow.

If `pyarrow==10.0.x`makes its way onto `condo-forge` in early December (as expected) we can do a patch release.

**Comment by gjoseph92** [comment]
Shuffle specifically we can expect to be slower (30% sounds about right to me) with queuing on by default.

If we're worried about versions of (transitive) dependencies having affected performance, @ncclementi maybe you should run the same A/B test, but turn off queuing for 0.2.0. If that looks the same/better, then we can attribute the wall-clock difference to queuing.

**Comment by ncclementi** [comment]
Here is the link to the AB test with no queuing: 

Wall time
![Screen Shot 2022-11-30 at 8 38 16 PM](https://user-images.githubusercontent.com/7526622/204945117-c30d1a38-cca4-4657-a6e1-d9b31f488627.png)

Memory
![Screen Shot 2022-11-30 at 8 39 23 PM](https://user-images.githubusercontent.com/7526622/204945255-1868371e-4522-4970-af91-56e4cc99100e.png)

**Comment by gjoseph92** [comment]
Hmm, so it looks like some changes are definitely not due to queuing. The H2O q9 change seems like the most important to investigate, but I wonder why all of H2O seems a little slower consistently.

**Comment by ncclementi** [comment]
I was looking into the specific case of q9 and I think the regression we see between 0.1.1 and the new 0.2.0 was introduced by dask/distributed at some point 

Notice that on latest q9-5GB was almost always averaging between 7-10 s (always using 2022.6.0), except the big bump when the version of dask was upgraded. 
![Screen Shot 2022-12-01 at 12 00 26 PM](https://user-images.githubusercontent.com/7526622/205113982-e0ac946b-e196-4b88-89ea-4fd6916c2766.png)

Now if look at upstream the earliest values we have are for 2022.7.01 and it's already higher by ~50% and this continues pretty consistently. See figures below

2022..7.1

![Screen Shot 2022-12-01 at 12 04 33 PM](https://user-images.githubusercontent.com/7526622/205114966-9727e6cd-7687-43fe-b3b1-f46bf7603b6e.png)

2022.9.2
![Screen Shot 2022-12-01 at 12 03 36 PM](https://user-images.githubusercontent.com/7526622/205114717-f80fb708-b2e2-4192-8ff9-bcd157e9f997.png)

We were back to the good times on 2022.10.0 but then back again to the regression.
![Screen Shot 2022-12-01 at 12 05 00 PM](https://user-images.githubusercontent.com/7526622/205115071-58227baa-39e7-4f22-a4a8-d63fce6a3a48.png)

In conclusion, this regression is being with us since 2022.7.1, it is probably worth investigating but it shouldn't block the release given all the other benefits that come with newer versions of dask.

**Comment by ncclementi** [comment]
> As we discussed -- running another A/B test that looks at bumping everything excepting the `fsspec` packages and then making a decision seems OK.

The results of this experiment did not have anything surprising. The behavior is the exact same as including the updated version of all the packages.

> Let's also confirm you can successfully complete a `conda build` with both of the build candidate, and the one you intend to test.

I was able to do this locally with no problems.

> Let's aim for a decision by midday tomorrow.
For regressions discussion see https://github.com/coiled/coiled-runtime/issues/526#issuecomment-1334094911
Got a PR open to kick this in https://github.com/coiled/coiled-runtime/pull/570

cc: @hayesgb

**Comment by hayesgb** [comment]
This looks good to me.  I would move forward with the newer release, and create an issue to investigate the Q9 issue separately.
